### ISSUE TITLE
storiesof-to-csf throws an error

### ISSUE THREAD

**Issue by mminkin** [storybookjs/storybook#17313]
Hi!
I'm trying to migrate lots of stories which still use the `storiesof` syntax to CSF. When I run the migration tool I get an error like so: 

```
npx -p @storybook/cli sb migrate storiesof-to-csf --glob=projects/src/group/Group.story.tsx                                    

=> Applying storiesof-to-csf: 1 files
npx: installed 236 in 17.104s
Processing 1 files...
Spawning 1 workers...
Sending 1 files to free worker...
[BABEL] Note: The code generator has deoptimised the styling of /Users/mminki/.npm/_npx/68609/lib/node_modules/@storybook/cli/node_modules/prettier/index.js as it exceeds the max of 500KB.
/Users/mminki/.npm/_npx/68609/lib/node_modules/@storybook/cli/node_modules/prettier/index.js:1036
});var shouldHighlight_1=shouldHighlight;var getChalk_1=getChalk;var _default$n=highlight;var jsTokensNs=_interopRequireWildcard$1(jsTokens);var _chalk=_interopRequireDefault$1(chalk);function _interopRequireDefault$1(obj){return obj&&obj.__esModule?obj:{default:obj};}function _getRequireWildcardCache$1(){if(typeof WeakMap!=="function")return null;var cache=new WeakMap();_getRequireWildcardCache$1=function _getRequireWildcardCache$1(){return cache;};return cache;}function _interopRequireWildcard$1(obj){if(obj&&obj.__esModule){return obj;}if(obj===null||_typeof2(obj)!=="object"&&typeof obj!=="function"){return{default:obj};}var cache=_getRequireWildcardCache$1();if(cache&&cache.has(obj)){return cache.get(obj);}var newObj={};var hasPropertyDescriptor=Object.defineProperty&&Object.getOwnPropertyDescriptor;for(var key in obj){if(Object.prototype.hasOwnProperty.call(obj,key)){var desc=hasPropertyDescriptor?Object.getOwnPropertyDescriptor(obj,key):null;if(desc&&(desc.get||desc.set)){Object.defineProperty(newObj,key,desc);}else{newObj[key]=obj[key];}}}newObj.default=obj;if(cache){cache.set(obj,newObj);}return newObj;}var sometimesKeywords=new Set(["as","async","from","get","of","set"]);function getDefs$1(chalk){return{keyword:chalk.cyan,capitalized:chalk.yellow,jsxIdentifier:chalk.yellow,punctuator:chalk.yellow,number:chalk.magenta,string:chalk.green,regex:chalk.magenta,comment:chalk.grey,invalid:chalk.white.bgRed.bold};}var NEWLINE$1=/\r\n|[\n\r\u2028\u2029]/;var BRACKET=/^[()[\]{}]$/;var tokenize;{var _matchToToken=jsTokensNs.matchToToken;var JSX_TAG=/^[a-z][\w-]*$/i;var getTokenType=function getTokenType(token,offset,text){if(token.type==="name"){if((0,lib$3.isKeyword)(token.value)||(0,lib$3.isStrictReservedWord)(token.value,true)||sometimesKeywords.has(token.value)){return"keyword";}if(JSX_TAG.test(token.value)&&(text[offset-1]==="<"||text.substr(offset-2,2)=="</")){return"jsxIdentifier";}if(token.value[0]!==token.value[0].toLowerCase()){return"capitalized";}}if(token.type==="punctuator"&&BRACKET.test(token.value)){return"bracket";}if(token.type==="invalid"&&(token.value==="@"||token.value==="#")){return"punctuator";}return token.type;};tokenize=/*#__PURE__*/regeneratorRuntime.mark(function tokenize(text){var match,_token;return regeneratorRuntime.wrap(function tokenize$(_context){while(1){switch(_context.prev=_context.next){case 0:if(!(match=jsTokensNs.default.exec(text))){_context.next=6;break;}_token=_matchToToken(match);_context.next=4;return{type:getTokenType(_token,match.index,text),value:_token.value};case 4:_context.next=0;break;case 6:case"end":return _context.stop();}}},tokenize);});}function highlightTokens(defs,text){var highlighted="";var _iterator32=_createForOfIteratorHelper(tokenize(text)),_step32;try{var _loop13=function _loop13(){var _step32$value=_step32.value,type=_step32$value.type,value=_step32$value.value;var colorize=defs[type];if(colorize){highlighted+=value.split(NEWLINE$1).map(function(str){return colorize(str);}).join("\n");}else{highlighted+=value;}};for(_iterator32.s();!(_step32=_iterator32.n()).done;){_loop13();}}catch(err){_iterator32.e(err);}finally{_iterator32.f();}return highlighted;}function shouldHighlight(options){return!!_chalk.default.supportsColor||options.forceColor;}function getChalk(options){return options.forceColor?new _chalk.default.constructor({enabled:true,level:1}):_chalk.default;}function highlight(code){var options=arguments.length>1&&arguments[1]!==undefined?arguments[1]:{};if(shouldHighlight(options)){var _chalk4=getChalk(options);var defs=getDefs$1(_chalk4);return highlightTokens(defs,code);}else{return code;}}var lib$2=/*#__PURE__*/Object.defineProperty({shouldHighlight:shouldHighlight_1,getChalk:getChalk_1,default:_default$n},'__esModule',{value:true});var codeFrameColumns_1=codeFrameColumns;var default_1=_default$m;var _highlight=_interopRequireWildcard(lib$2);function _getRequireWildcardCache(){if(typeof WeakMap!=="function")return null;var cache=new WeakMap();_getRequireWildcardCache=function _getRequireWildcardCache(){return cache;};return cache;}function _interopRequireWildcard(obj){if(obj&&obj.__esModule){return obj;}if(obj===null||_typeof2(obj)!=="object"&&typeof obj!=="function"){return{default:obj};}var cache=_getRequireWildcardCache();if(cache&&cache.has(obj)){return cache.get(obj);}var newObj={};var hasPropertyDescriptor=Object.defineProperty&&Object.getOwnPropertyDescriptor;for(var key in obj){if(Object.prototype.hasOwnProperty.call(obj,key)){var desc=hasPropertyDescriptor?Object.getOwnPropertyDescriptor(obj,key):null;if(desc&&(desc.get||desc.set)){Object.defineProperty(newObj,key,desc);}else{newObj[key]=obj[key];}}}newObj.default=obj;if(cache){cache.set(obj,newObj);}return newObj;}var deprecationWarningShown=false;function getDefs(chalk){return{gutter:chalk.grey,marker:chalk.red.bold,message:chalk.red.bold};}var NEWLINE=/\r\n|[\n\r\u2028\u2029]/;function getMarkerLines(loc,source,opts){var startLoc=Object.assign({column:0,line:-1},loc.start);var endLoc=Object.assign({},startLoc,loc.end);var _ref31=opts||{},_ref31$linesAbove=_ref31.linesAbove,linesAbove=_ref31$linesAbove===void 0?2:_ref31$linesAbove,_ref31$linesBelow=_ref31.linesBelow,linesBelow=_ref31$linesBelow===void 0?3:_ref31$linesBelow;var startLine=startLoc.line;var startColumn=startLoc.column;var endLine=endLoc.line;var endColumn=endLoc.column;var start=Math.max(startLine-(linesAbove+1),0);var end=Math.min(source.length,endLine+linesBelow);if(startLine===-1){start=0;}if(endLine===-1){end=source.length;}var lineDiff=endLine-startLine;var markerLines={};if(lineDiff){for(var i=0;i<=lineDiff;i++){var lineNumber=i+startLine;if(!startColumn){markerLines[lineNumber]=true;}else if(i===0){var sourceLength=source[lineNumber-1].length;markerLines[lineNumber]=[startColumn,sourceLength-startColumn+1];}else if(i===lineDiff){markerLines[lineNumber]=[0,endColumn];}else{var _sourceLength=source[lineNumber-i].length;markerLines[lineNumber]=[0,_sourceLength];}}}else{if(startColumn===endColumn){if(startColumn){markerLines[startLine]=[startColumn,0];}else{markerLines[startLine]=true;}}else{markerLines[startLine]=[startColumn,endColumn-startColumn];}}return{start:start,end:end,markerLines:markerLines};}function codeFrameColumns(rawLines,loc){var opts=arguments.length>2&&arguments[2]!==undefined?arguments[2]:{};var highlighted=(opts.highlightCode||opts.forceColor)&&(0,_highlight.shouldHighlight)(opts);var chalk=(0,_highlight.getChalk)(opts);var defs=getDefs(chalk);var maybeHighlight=function maybeHighlight(chalkFn,string){return highlighted?chalkFn(string):string;};var lines=rawLines.split(NEWLINE);var _getMarkerLines=getMarkerLines(loc,lines,opts),start=_getMarkerLines.start,end=_getMarkerLines.end,markerLines=_getMarkerLines.markerLines;var hasColumns=loc.start&&typeof loc.start.column==="number";var numberMaxWidth=String(end).length;var highlightedLines=highlighted?(0,_highlight.default)(rawLines,opts):rawLines;var frame=highlightedLines.split(NEWLINE).slice(start,end).map(function(line,index){var number=start+1+index;var paddedNumber=" ".concat(number).slice(-numberMaxWidth);var gutter=" ".concat(paddedNumber," |");var hasMarker=markerLines[number];var lastMarkerLine=!markerLines[number+1];if(hasMarker){var markerLine="";if(Array.isArray(hasMarker)){var markerSpacing=line.slice(0,Math.max(hasMarker[0]-1,0)).replace(/[^\t]/g," ");var numberOfMarkers=hasMarker[1]||1;markerLine=["\n ",maybeHighlight(defs.gutter,gutter.replace(/\d/g," "))," ",markerSpacing,maybeHighlight(defs.marker,"^").repeat(numberOfMarkers)].join("");if(lastMarkerLine&&opts.message){markerLine+=" "+maybeHighlight(defs.message,opts.message);}}return[maybeHighlight(defs.marker,">"),maybeHighlight(defs.gutter,gutter),line.length>0?" ".concat(line):"",markerLine].join("");}else{return" ".concat(maybeHighlight(defs.gutter,gutter)).concat(line.length>0?" ".concat(line):"");}}).join("\n");if(opts.message&&!hasColumns){frame="".concat(" ".repeat(numberMaxWidth+1)).concat(opts.message,"\n").concat(frame);}if(highlighted){return chalk.reset(frame);}else{return frame;}}function _default$m(rawLines,lineNumber,colNumber){var opts=arguments.length>3&&arguments[3]!==undefined?arguments[3]:{};if(!deprecationWarningShown){deprecationWarningShown=true;var message="Passing lineNumber and colNumber is deprecated to @babel/code-frame. Please use `codeFrameColumns`.";if(process.emitWarning){process.emitWarning(message,"DeprecationWarning");}else{var deprecationError=new Error(message);deprecationError.name="DeprecationWarning";console.warn(new Error(message));}}colNumber=Math.max(colNumber,0);var location={start:{column:colNumber,line:lineNumber}};return codeFrameColumns(rawLines,location,opts);}var lib$1=/*#__PURE__*/Object.defineProperty({codeFrameColumns:codeFrameColumns_1,default:default_1},'__esModule',{value:true});var ConfigError=errors.ConfigError;var locStart$o=loc$6.locStart,locEnd$n=loc$6.locEnd;// Use defineProperties()/getOwnPropertyDescriptor() to prevent


ReferenceError: regeneratorRuntime is not defined
    at Object.<anonymous> (/Users/mminki/.npm/_npx/68609/lib/node_modules/@storybook/cli/node_modules/prettier/index.js:14531:3)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module._compile (/Users/mminki/.npm/_npx/68757/lib/node_modules/jscodeshift/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.newLoader [as .js] (/Users/mminki/.npm/_npx/68757/lib/node_modules/jscodeshift/node_modules/pirates/lib/index.js:141:7)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mminki/.npm/_npx/68609/lib/node_modules/@storybook/cli/node_modules/@storybook/codemod/dist/cjs/transforms/storiesof-to-csf.js:8:40)
All done.
Results:
0 errors
0 unmodified
0 skipped
0 ok
Time elapsed: 19.156seconds
```

Environment Info:
```
  System:
    OS: macOS 12.0.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 14.15.5 - ~/.nvm/versions/node/v14.15.5/bin/node
    Yarn: 1.22.10 - ~/work/Modern Client/yammer-clients/node_modules/.bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v14.15.5/bin/npm
  Browsers:
    Chrome: 98.0.4758.66
    Edge: 97.0.1072.69
    Firefox: 92.0
    Safari: 15.1
  npmPackages:
    @storybook/addon-a11y: 6.3.12 => 6.3.12
    @storybook/addon-actions: 6.3.12 => 6.3.12
    @storybook/addon-docs: 6.3.12 => 6.3.12
    @storybook/addon-knobs: 6.3.1 => 6.3.1
    @storybook/addon-storyshots: 6.3.12 => 6.3.12
    @storybook/builder-webpack5: 6.3.12 => 6.3.12
    @storybook/manager-webpack5: 6.3.12 => 6.3.12
    @storybook/react: 6.3.12 => 6.3.12
    @storybook/theming: 6.3.12 => 6.3.12
```

When I googled this I haven't found anyone with the same issue. Help?

**Comment by shilman** [comment]
It's a dependency of `@storybook/codemod` which is the code that's running. Perhaps we also need to add it to `@storybook/cli`? As a workaround, try adding this to your project dev dependencies

```
    "regenerator-runtime": "^0.13.7"
```

**Comment by darekkay** [comment]
In my case it was #16933 that caused the error, i.e., a babel config file in the root of the project. I have temporally removed the `babel.config.json` file and the codemod ran without errors.

**Comment by ndelangen** [comment]
Fixed in 7.0 beta
